src/transaction/TransactionController.ts

```diff
@@ -117,6 +133,15 @@ export enum WalletDevice {
   OTHER = 'other_device',
 }
 
+/**
+ * Source of data used to reconcile local transactions end state
+ */
+export enum StateReconcileMethod {
+  ETHERSCAN = 'etherscan',
+  BLOCKCHAIN = 'blockchain',
+  OTHER = 'other',
```

Comment:
[Q] What does other potentially refer to?

Reply:
For the next release I'll develop another reconciliation logic using the Blockchain as a source of truth. The `OTHER` can be removed, but it could be used as a default.